libsmartcols/src/filter-param.c

```diff
@@ -110,9 +111,14 @@ struct filter_node *filter_new_param(
 		return NULL;
 	}
 
+	n->percentage_holder = 0;
 	if (holder == F_HOLDER_COLUMN) {
+		size_t len;
 		n->holder_name = strdup((char *) data);
 		DBG(FLTR, ul_debugobj(fltr, "new %s holder", n->holder_name));
+		len = strlen(n->holder_name);
+		if (len > 1 && n->holder_name[len - 1] == '%')
+			n->percentage_holder = 1;
```

Comment:
Maybe you can use
```
 n->percentage_holder = endswith(n->holder_name, "%") ? 1 : 0;
```
to keep it simple :)

Reply:
(Fixed locally).